harness/assert.js

```diff
@@ -84,7 +84,7 @@ assert.throws = function (expectedErrorConstructor, func, message) {
       message += 'Thrown value was not an object!';
       $ERROR(message);
     } else if (thrown.constructor !== expectedErrorConstructor) {
-      message += 'Expected a ' + expectedErrorConstructor.name + ' but got a ' + thrown.constructor.name;
+      message += 'Expected a ' + expectedErrorConstructor.name + ' but got a ' + thrown.constructor.name + ' ' + thrown.message;
```

Comment:
I might be wrong but IIRC we've had a reason to not include message here.

Anyway, the fact test runners might try to parse this line for the whatever reason, I'm not sure if this would require a new version. One quick workaround would be putting this in a new line and handle if message is not undefined.

This is probably too much of a nit pick.

Reply:
Hmm, so should I revert this change? It made confirmation better for me, but I'm not married to it.